scripts/vtk_convert.py

```diff
@@ -0,0 +1,91 @@
+from vtk import vtkPolyDataReader, vtkStructuredPointsReader, \
+    vtkXMLImageDataWriter, vtkXMLPolyDataWriter
+from girder_client import GirderClient
+import os
+import click
+import re
+
+@click.command()
+@click.option('--inputfolderid', required=True, prompt=True, help='Source Folder ID')
```

Comment:
I think `prompt=True` is a good feature for the API key, since users may not want to put it into the command, since it would cause a password to be added to their shell history. However, I don't think we need to prompt for other missing options; we can just error out and expect the user to run the command with them.